src/Streamly/Internal/Data/Fold.hs

```diff
@@ -65,8 +65,9 @@ module Streamly.Internal.Data.Fold
 
     -- ** Collectors
     , toList
-    , toListRevF  -- experimental
-    -- $toListRevF
+    , toListRev  -- experimental
+    , toStream
```

Comment:
Add the `-- $toListRev` note here as before, it is a haddock reference to show the labeled note.